Validation/RecoTrack/python/PostProcessorTracker_cfi.py

```diff
@@ -348,11 +349,21 @@ def _addNoFlow(module):
 displacedTrackValidation.toReplaceWith(postProcessorTrack,postProcessorTrackDisplaced)
 displacedTrackValidation.toReplaceWith(postProcessorTrackSummary,postProcessorTrackSummaryDisplaced)
 
+from Configuration.ProcessModifiers.pp_on_AA_cff import pp_on_AA
+from Configuration.Eras.Modifier_pp_on_PbPb_run3_cff import pp_on_PbPb_run3
+
+_defaultSubdirsHIon =  _defaultSubdirs + ["Tracking/HIPixelTrack/*"]
+_defaultSubdirsSummaryHIon = [e.replace("/*","") for e in _defaultSubdirsHIon]
+
+(pp_on_AA | pp_on_PbPb_run3).toModify(postProcessorTrack,subDirs = _defaultSubdirsHIon)
```

Comment:
This would be equivalent to `pp_on_AA` by itself.
Both the Run 2 and Run 3 variants already contain the process modifier `pp_on_AA`

Reply:
Fixed, thanks.